src/app/pages/event/create.vue

```diff
@@ -16,9 +15,17 @@ const store = useStore()
 
 // data
 const title = t('title')
+const { setTitle } = useHeaderTitle()
 
 // initialization
 useHeadDefault({ title })
+
+onMounted(() => {
+  setTitle(title)
+})
+onBeforeUnmount(() => {
+  setTitle('')
+})
```

Comment:
`onMounted` and `onBeforeUnmount` can be moved into the composable so only this remains:
```suggestion
useHeaderTitle(title)

// initialization
useHeadDefault({ title })
```